src/scripts/trainers/Trainer.ts

```diff
@@ -2,19 +2,25 @@
 class Trainer {
 
     public name: string;
+    team: GymPokemon[];
 
     constructor(
         public trainerClass: string,
-        public team: GymPokemon[],
+        team: GymPokemon[],
         name?: string,
         public subTrainerClass?: string
     ) {
         this.name = name ? `${trainerClass} ${name}` : trainerClass;
+        this.team = team;
```

Comment:
Does this part need to be changed?
As far as I can tell, it seems to be doing the same thing in long form?

Reply:
I thought local parameters defaulted to private (just looked it up, it does not). But you can do the same public-in-constructor trick, just with private. I think i have made this mistake a couple of places... Oh well, i learned something now, and i will fix the others, if i come by them.
But yeah i want it to be private, because i don't think anyone should ever use the team, without it being filtered by requirements first